tbx/project_styleguide/templates/patterns/atoms/avatar/avatar.html

```diff
@@ -7,6 +7,6 @@
     <picture>
         <source media="(max-width: 598px)" srcset="{{  avatar_mobile.url }} 1x, {{ avatar_desktop.url }} 2x" />
         <source media="(min-width: 599px)" srcset="{{ avatar_desktop.url }} 1x, {{ avatar_desktop_retina.url }} 2x" />
-        <img src="{{ avatar_desktop.url }}" alt="{{ avatar_desktop.alt }}" {% if lazy_load %}loading="lazy"{% endif %} class="avatar__image" />
+        <img src="{{ avatar_desktop.url }}" alt="" {% if lazy_load %}loading="lazy"{% endif %} class="avatar__image" />
```

Comment:
The example I tested had a terrible alt text for the author image - changed to empty and made more explicit in the author template instead.

Reply:
Checking we're ok that all avatars will be decorative?

Reply:
I've done a quick check of everywhere we use it so far, and there is always a name alongside it. I did spot an issue with the styleguide yaml which I've updated in 3c181f3002cba68c41414e9b2ad20dc92cf6d812